ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -120,7 +120,11 @@ export const WorkflowNodeSummary = (props: Props) => {
                         .map(link => (
                             <a
                                 className='argo-button argo-button--base-o'
-                                href={link.url.replace('${metadata.namespace}', props.workflow.metadata.namespace).replace('${metadata.name}', props.node.id)}>
+                                href={link.url
+                                    .replace('${metadata.namespace}', props.workflow.metadata.namespace)
+                                    .replace('${metadata.name}', props.node.id)
+                                    .replace('${status.startedAt}', props.workflow.status.startedAt)
```

Comment:
should this be the started finished at dates of the node not the workflow?

Reply:
thx! I fixed!